pkg/cmd/server/api/v1/conversions.go

```diff
@@ -27,7 +27,7 @@ func SetDefaults_MasterConfig(obj *MasterConfig) {
 			election.LockNamespace = "kube-system"
 		}
 		if len(election.LockResource.Group) == 0 && len(election.LockResource.Resource) == 0 {
-			election.LockResource.Resource = "endpoints"
+			election.LockResource.Resource = "configmaps"
```

Comment:
I assume some watches or controllers cause CPU churn with `endpoints`?  What makes `configmaps` a good choice in this case?  The lock just uses annotations so any resource would do.